ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/answers/activity-file-answer/activityFileAnswer.component.ts

```diff
@@ -52,37 +79,127 @@ export class ActivityFileAnswer implements OnDestroy {
                 .pipe(
                     catchError(err => {
                         this.logger.logDebug('ActivityFileAnswer getUploadUrl error:', err);
+                        this.errorMessage = err.message;
```

Comment:
We should clear `errorMessage` whenever a new file is selected
1. We select a file that is too big
2. UI shows error "File is too big" (or something like that)
3. We select a file that is of allowed size.
4. UI keeps showing "File is too big" error.

Reply:
My bad. 
Fixed at 8d677f21d8cc286f7d0db97f4f7ce19197a03dcf